public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -770,15 +790,19 @@ function saveBookmark() {
         sessionStorage.setItem("Bookmark", URL);
         createBookmarks();
     }
-    else {
-        console.log("one bookmark");
-    }
 }
 function createBookmarks() {
     const LI = document.createElement("li");
     const URL = sessionStorage.getItem("Bookmark");
     const TITLE = sessionStorage.getItem("Title");
-    let INNER = `<div> <p>Bookmark: <a style="display: unset;" href="${URL}"> ${TITLE}</a></p></div>`;
-    LI.innerHTML = INNER;
-    URL ? document.querySelector("#bm-list")?.appendChild(LI) : console.log("No bookmarks");
+    LI.id = "sbBookmark";
+    LI.innerHTML = `<div > <p><a style="display: unset;" href="${URL}"> ${TITLE}</a><a id="removeBookmark" style="display: unset;" href="#0" onclick="removeBookmarks()">| Remove</a> </p></div>`;
```

Comment:
The remove bookmark link is active and does do a soft page redirect (which is unnecessary and undesired). Either remove the href or apply some sort of e.preventDefault().